Chat/AppManager.cpp

```diff
@@ -0,0 +1,23 @@
+#include "AppManager.h"
+
+void AppManager::Work()
+{
+    network_.PrepareNetwork();
+    std::thread network_loop(Network::StartNetwork, &network_);
```

Comment:
To make it harder, you shouldn't use threads or mutex from library! Only WINAPI such as CreateThread, CreateMutex, Release, CloseHandle etc.